labelCloud/model/point_cloud.py

```diff
@@ -210,6 +226,30 @@ def label_counts(self) -> Optional[Dict[str, int]]:
             return counter
         return None
 
+    @property
+    def has_label(self) -> bool:
+        return self.labels is not None
+
+    def update_colors_selected_points(
+        self, points_inside: npt.NDArray[np.bool_]
+    ) -> None:
+        GL.glBindBuffer(GL.GL_ARRAY_BUFFER, self.label_vbo)
+        inside_idx = np.where(points_inside)[0]
+        if inside_idx.shape[0] == 0:
+            logging.warning("No points are found inside the selected boxes.")
+            return
+        logging.info(f"Update {len(inside_idx)} point colors in label VBO.")
```

Comment:
I think most users don't know a "VBO", so maybe sth. like "Labeled xyz points inside the current bounding box with label xyz."

Reply:
I changed the logging level to debug as strictly speaking this method doesn't change the labels but only the label colors in gpu. Instead I added another logging in higher level where it does both here [5e7ed76](https://github.com/ch-sa/labelCloud/pull/104/commits/5e7ed76d5c65cede0790936f7fc56a995210268a)